Cargo.toml

```diff
@@ -18,9 +18,14 @@ page_size = "0.4"
 signal-hook = "0.3"
 serde_cbor = "0.11"
 eif_loader = { path = "./eif_loader" }
+eif_defs = { path = "./eif_defs" }
 enclave_build = { path = "./enclave_build" }
 openssl = "0.10"
 vsock = "0.1.5"
+sha2 = "0.9.5"
+hex = "0.3.2"
+aws-nitro-enclaves-cose = "0.1"
+tokio = { version = "1.0", features = ["rt-multi-thread"] }
```

Comment:
I would rather avoid bringing this heavy dependency on tokio.

Reply:
I removed the dependency.